src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -143,6 +143,30 @@ case class SubmissionReport(
   notstarted: Seq[SubmissionValidationEntityInputs]
 )
 
+case class CallMetadata(
+  inputs: Map[String, String],
+  status: String,
+  outputs: Option[Map[String, String]],
+  start: Option[DateTime],
+  end: Option[DateTime],
+  jobid: Option[String],
+  rc: Option[Int],
```

Comment:
That would break the unmarshaling from Cromwell who have chosen the name "rc".
You could ask the Cromwell team to rename according to your tastes, and I'll then make the corresponding changes.
Or we could have pair of case classes, CallMetadataCromwellian and CallMetadataVoetesque (and similarly for ExecutionMetadata), and I'll map from one to the other so that we can do arbitrary renaming of Cromwell's fields.
Let me know what you'd prefer.